pkg/controller/node/nodecontroller_test.go

```diff
@@ -1726,9 +1726,6 @@ func TestMonitorNodeStatusMarkPodsNotReady(t *testing.T) {
 							CreationTimestamp: metav1.Date(2012, 1, 1, 0, 0, 0, 0, time.UTC),
 						},
 						Status: v1.NodeStatus{
-							NodeInfo: v1.NodeSystemInfo{
-								KubeletVersion: "v1.2.0",
```

Comment:
@gmarek - removing this tricks the nc on the kubelet version, when it can't be parsed it assumes that version is < 1.2.0, hence taking codepath for outdated kubelet ?

Reply:
Yes, you're right. This means that we need to keep this until we remove outdated logic.

Reply:
yup indeed.

Reply:
@gmarek - yup, if we remove outdated logic, tests wont pass. 

we should get them out of this PR, then add them with the PR that removes the outdated logic

Reply:
Can we just keep this 'KubeletVersion' setting in the test for the time being?